j2735_v2x_msgs/CMakeLists.txt

```diff
@@ -25,15 +25,16 @@ find_package(ros_environment REQUIRED)
 
 set(ROS_VERSION $ENV{ROS_VERSION})
 
+file(GLOB msg_files msg/*.msg)
+set(msgs )
```

Comment:
Nit: Sorry I missed this in the first round. I think we can remove `set(msgs )`, which I think is only there to declare the variable empty and we may not need to do that. Could you check if we can build it fine wihout it? I'll approve it though